crates/swc/tests/fixture/issues-6xxx/6418/1/output/input.js

```diff
@@ -1,2 +1,2 @@
-var foo = function o() {};
+var foo = function foo() {};
```

Comment:
I'm not sure about this change..
I think this is caused by the type change of isModule.